app/src/main/res/layout/fragment_videos_list_item.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:padding="8dp"
+    >
+
+
+  <VideoView
+      android:id="@+id/app_video"
+      android:layout_width="match_parent"
+      android:layout_height="200dp"
+      android:orientation="vertical"
+      android:layout_gravity="center"
+      />
+
+  <android.support.design.widget.CoordinatorLayout
+      android:id="@+id/app_info_bar"
+      android:layout_width="match_parent"
+      android:layout_height="80dp"
+      android:orientation="horizontal">
+
+    <ImageView
+        android:id="@+id/discovery_app_icon"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_gravity="left|center"
+        android:src="@drawable/aptoide_gradient_rounded"
+        android:layout_marginLeft="16dp"
+        />
+
+      <TextView
+          android:id="@+id/app_title"
+          android:text="Game #1 - Verrry noice"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          app:layout_anchor="@+id/discovery_app_icon"
+          app:layout_anchorGravity="right"
+          android:layout_gravity="center|right"
+          android:layout_marginLeft="70dp"
+          android:layout_marginTop="15dp"
+          android:textColor="@color/white"
+          />
+
+      <ImageView
+          android:id="@+id/rating_star"
+          android:layout_width="10dp"
+          android:layout_height="10dp"
+          app:layout_anchor="@+id/discovery_app_icon"
+          app:layout_anchorGravity="bottom|right"
+          android:layout_marginLeft="70dp"
+          android:layout_marginBottom="30dp"
+          android:src="@drawable/ic_star_white"
+          />
+
+      <TextView
+          android:id="@+id/app_score"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:text="0.0"
+          app:layout_anchor="@+id/rating_star"
+          app:layout_anchorGravity="right|center"
+          android:layout_gravity="right"
+          android:layout_marginTop="36dp"
+          android:layout_marginLeft="83dp"
+          android:textColor="@color/white"
+          />
+
+    <Button
+        android:id="@+id/install_button"
+        android:layout_width="83dp"
+        android:layout_height="32dp"
+        android:text="INSTALL"
+        android:layout_gravity="right|center"
+        android:layout_marginRight="16dp"
+        android:background="@drawable/rounded_corners_white"
+        />
+
+  </android.support.design.widget.CoordinatorLayout>
+
+</LinearLayout>
```

Comment:
This xml needs a revision to use all the Aptoide styles.

Reply:
fixed